### PR TITLE
fix: @typescript-eslint/no-floating-promises

### PR DESCRIPTION
**Pull request by sdstolworthy**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sobolk — DISMISSED** [review]

**Comment by sobolk** [comment]
please run e2e.

**Review by Amplifiyer — APPROVED** [review]

**Review by sobolk — APPROVED** [review]

File: packages/amplify-category-geo/src/provider-controllers/index.ts [hunk 1]
```diff
@@ -6,9 +6,7 @@ import { provider, ServiceName } from '../service-utils/constants';
 import { checkGeoResourceExists } from '../service-utils/resourceUtils';
 import { getServiceFriendlyName } from '../service-walkthroughs/resourceWalkthrough';
 import { addGeofenceCollectionResource, removeGeofenceCollectionResource, updateGeofenceCollectionResource } from './geofenceCollection';
-import {
-  addMapResource, addMapResourceHeadless, removeMapResource, updateMapResource, updateMapResourceHeadless,
-} from './map';
+import { addMapResource, addMapResourceHeadless, removeMapResource, updateMapResource, updateMapResourceHeadless } from './map';
```
Comment (evcodes) [inline packages/amplify-category-geo/src/provider-controllers/index.ts:9]: how come the imports in this file aren't separated by new lines?

File: packages/amplify-category-notifications/src/pinpoint-helper.ts [hunk 2]
```diff
@@ -13,11 +13,7 @@ import {
   AmplifyFault,
 } from 'amplify-cli-core';
 import { printer } from 'amplify-prompts';
-import {
-  invokeAnalyticsAPICreateResource,
-  invokeAnalyticsAPIGetResources,
-  invokeAnalyticsPush,
-} from './plugin-client-api-analytics';
+import { invokeAnalyticsAPICreateResource, invokeAnalyticsAPIGetResources, invokeAnalyticsPush } from './plugin-client-api-analytics';
```
Comment (evcodes) [inline packages/amplify-category-notifications/src/pinpoint-helper.ts:16]: same question here

File: packages/amplify-cli/src/amplify-exception-handler.ts [hunk 3]
```diff
@@ -61,18 +51,22 @@ export const handleException = async (exception: unknown): Promise<void> => {
   }
 
   await executeSafely(
-    () => executeHooks(HooksMeta.getInstance(undefined, 'post', {
-      message: deepestException.message ?? 'undefined error in Amplify process',
-      stack: deepestException.stack ?? 'undefined error stack',
-    })),
+    () =>
+      executeHooks(
```
Comment (evcodes) [inline packages/amplify-cli/src/amplify-exception-handler.ts:55]: I think this should be awaited?

File: packages/amplify-provider-awscloudformation/src/push-resources.ts [hunk 4]
```diff
@@ -431,16 +432,21 @@ export const run = async (context: $TSContext, resourceDefinition: $TSObject, re
     await downloadAPIModels(context, newAPIresources);
 
     // remove ephemeral Lambda layer state
-    if (resources.concat(resourcesToBeDeleted).filter((r: { service: string; }) => r.service === AmplifySupportedService.LAMBDA_LAYER).length > 0) {
+    if (
+      resources.concat(resourcesToBeDeleted).filter((r: { service: string }) => r.service === AmplifySupportedService.LAMBDA_LAYER).length >
+      0
+    ) {
```
Comment (evcodes) [inline packages/amplify-provider-awscloudformation/src/push-resources.ts:438]: This formatting is a bit weird

**Review by evcodes — APPROVED** [review]
a few nits, none of which are blocking the PR

**Pull request merged**